autoload/fzf_tags.vim

```diff
@@ -32,10 +32,13 @@ function! fzf_tags#Find(identifier)
     execute 'tag' identifier
   else
     let expect_keys = join(keys(s:actions), ',')
+    if !exists('g:fzf_tags_prompt')
```

Comment:
As you're planning to add more customizations, this makes more sense at the top of the file, just before `let s:actions...`.

Reply:
@zackhsi It's good idea. Thank you.
I fixed it. Please check it out!